.github/workflows/trusted-cd-mac-release.yml

```diff
@@ -17,14 +17,15 @@ jobs:
       - name: "Installing dependencies"
         run: |
           rm '/usr/local/bin/2to3'
-          brew install llvm@13
+          brew uninstall llvm@13
```

Comment:
This one has an explicit uninstall of LLVM 13, but the ci-mac-on-main-pr.yml just installs 14. Is it necessary to have this uninstall here, and if so, why isn't it needed for the CI path?